chef_master/source/_templates/nav-docs.html

```diff
@@ -126,8 +131,44 @@
           },
           {
             "title": "Microsoft Windows Guide",
-            "hasSubItems": false,
-            "url": "/windows.html"
+            "hasSubItems": true,
+            "subItems": [
+              {
+                "title": "Microsoft Azure",
```

Comment:
I'm not sure if we consider Azure to be a Windows things. Lots of people are running almost exclusively Linux workloads there.

Reply:
Maybe a "Chef on Windows" guide? As this seems to be Windows specific here.

Reply:
Went with @stuartpreston's "Chef on Windows Guide". I don't have a better name